docs/_docs/collections.md

```diff
@@ -125,7 +125,7 @@ help you control the output url for the entire collection.
 
 ## Custom Sorting of Documents
 
-By default, documents in a collection are sorted by their paths. But you can control this sorting via the collection's metadata.
+By default, documents in a collection are sorted by their `date`s (if both have that front-matter key) or by their paths (if one or both do not have the `date` key). But you can control this sorting via the collection's metadata.
```

Comment:
How about being a little more verbose? What say?

```suggestion
By default, two documents in a collection are sorted by their `date` attribute when both of them have the `date` key in their front matter. However, if either or both documents do not have the `date` key in their front matter, they are sorted by their respective paths.

You can control this sorting via the collection's metadata:
```

Reply:
Sounds good!

Reply:
It's more clear now. Good job!